plugin/transfer/notifiy.go

```diff
@@ -0,0 +1,73 @@
+package transfer
+
+import (
+	"fmt"
+
+	"github.com/coredns/coredns/plugin/pkg/rcode"
+	"github.com/miekg/dns"
+)
+
+// Notify sends notifies for incoming messages from plugins
+func (t Transfer) Notify(ch <-chan []string) {
+	for zones := range ch {
+		for _, zone := range zones {
+			x := t.getInstanceForZone(zone)
+			notify(zone, x.to)
+		}
+	}
+}
+
+func (t Transfer) getInstanceForZone(zone string) *xfr {
```

Comment:
can you merge this with the notify function? Maybe name it, `notifyForZone` or something?